app/views/course_templates/index.html.erb

```diff
@@ -19,11 +19,11 @@
         <td><%= course_template.name %></td>
         <td><%= course_template.title %></td>
         <td><%= course_template.description %></td>
-        <td><%= course_template.material_url %></td>
-        <td><%= course_template.source_url %></td>
+        <td><%= link_to course_template.material_url, course_template.material_url %></td>
+        <td><%= link_to course_template.source_url, course_template.source_url %></td>
```

Comment:
Links are fairly easy to copy in most browsers, but I agree in that it's a bit silly to have a link that one almost never ought to left-click.